compiler/rustc_error_codes/src/error_codes/E0793.md

```diff
@@ -0,0 +1,49 @@
+A lifetime parameter of a function definition is called *late-bound* if it both:
+
+1. appears in an argument type
```

Comment:
Please try to follow the formatting of error code:

```
[small explanation (one/two sentence)]

Erroneous code example:

[erroneous code example]

[long explanation]

[fixed code example]
```

Reply:
@GuillaumeGomez
I refactored it now. Is the last bit of information OK, or should I merge it in [long explanation]?

Reply:
It's fine like this, thanks.